ios/sentry.properties

```diff
@@ -1,4 +1,4 @@
 defaults.url=https://sentry.io/
 defaults.org=testorg-az
-defaults.project=mobile-webinar
-auth.token=XXXX
+defaults.project=adamreactnative
+auth.token=bbece28fc4b847baaed54aa3518c5e7ced9bbaaac4404fc48f81dea11347caae
```

Comment:
@sentryadam0000345 Remove this Auth Token from source code, then delete it from your Sentry.io settings and generate a new one and start using that.

Reply:
Done